gossip/hub/configpb/config.proto

```diff
@@ -0,0 +1,74 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package configpb;
+
+import "github.com/google/trillian/crypto/keyspb/keyspb.proto";
+import "github.com/google/trillian/crypto/sigpb/sigpb.proto";
+import "google/protobuf/any.proto";
+
+// HubMultiConfig wraps up a HubBackendSet and corresponding HubConfigSet so
+// that they can easily be parsed as a single proto.
+message HubMultiConfig {
+  // The set of backends that this configuration will use to send requests to.
+  // The names of the backends in the HubBackendSet must all be distinct.
+  HubBackendSet backends = 1;
+  // The set oubs that will use the above backends. All the protos in this
```

Comment:
of hubs?

Reply:
Fixed.

Reply:
doesn't look like the fix got pushed.

Reply:
Looks like it's there in a fixup commit to me: https://github.com/google/trillian-examples/pull/27/commits/58988380bf06cedd4aad5d836682df52aac83a5b -- does that not show up for you?

Reply:
Now it's showing up